### ISSUE TITLE
Can't get more FPS

### ISSUE THREAD

**Issue by vraixen** [34736384/genshin-fps-unlock#204]
Do you know why i can't get more then 80/90 fps? i'm running the game in 4k 
my specs are RTX 4080, 48GB Ram 3000mhz, i7-10700k. Nothing is fully utilized... not even a single core
![immagine](https://github.com/34736384/genshin-fps-unlock/assets/58710194/b9ab988d-071b-40fe-b6e8-9e72f6a5bc07)
Maybe it's just the game? because when i used to play in 1080p i was able to get 120fps with 2080 super.

**Comment by AquaSimulacra** [comment]
Genshin Impact is a game demanding a powerful CPU. I replaced my 10700K with 13600K, resulting in a big improvement in FPS.

**Comment by vraixen** [comment]
> 

but it doesn't make much sense since is not fully utilizing the CPU or a single core.. my assumption is that this is a limition of DIrectx 11 which has a limited amount of draw calls available compared to Directx 12....

**Comment by AquaSimulacra** [comment]
No, actually it is because the trash optimization of Genshin Impact which cannot utilize multicore of CPU.

**Comment by vraixen** [comment]
0

> 

yeah but what you says doesn't make much sense considering that not a single one of my cores is fully utilized.. as i said i suspect is limited by the draw calls

**Comment by HadouGun** [comment]
There's also Core parking, have you unparked your cores yet?
Update Motherboard Bios
Overclock CPU/Undervolt CPU
Overclock Graphics Card/Undervolt Graphics Card
Enable XMP/Expo
Debloat Windows (TechYesCity has a good tutorial for 2023)

**Comment by HadouGun** [comment]
> Genshin Impact is a game demanding a powerful CPU. I replaced my 10700K with 13600K, resulting in a big improvement in FPS. If you want to run smoothly, I suggest you replace your CPU with Intel 13th or wait for the 14th. (I am not from the marketing team of Intel)

I've been looking at the 7800X3D it seems like the best option atm since Intel is going to change platforms soon and it's specifically designed for games, yet Genshin we don't know what runs better because nobody benchmarks it (because they'd have to use this mod to do so).

**Comment by HadouGun** [comment]
With a 9700k at 4.9Ghz, 3200Mhz Ram and a 4090 with a +200 Core + 1000 Memory OC I get 100-120 @4k.
I have Afterburner up to maintain OC on graphics and I'm using Resolution timer .5 not sure how much that changes things.

**Comment by Raze876** [comment]
I have a 11700F(all core 4.4)/3080 10gb/4x8 3200 cl16/game installed on gen 4 nvme/ I play Genshin maxed @1440p and 720p and I know in Sumeru both that stairs area at the adventurers guild and at the crafting bench a little further ahead drop the fps for me to even like 80s, gpu usage around 55% and rivatuner OSD showing my highest cpu core in high 80s, maybe a mix of cpu bottleneck and some other limitation. 

I believe the cpu usage shown is overall though as if im doing something else to max my cpu out while playing the osd will show all my cores at about 98-100% so maybe some areas have some other limitation/bad optimization.

**Comment by HadouGun** [comment]
Inside the water in Fontaine I've had no issues also the fps is smooth in the main city.

**Comment by BlueStarBRS** [comment]
you need a better cpu with more power.

**Comment by fai2523** [comment]
> 0
> 
> > 
> 
> yeah but what you says doesn't make much sense considering that not a single one of my cores is fully utilized.. as i said i suspect is limited by the draw calls

None of my core are being maxed either playing on 5800x3d but I got 120+ FPS in the same area. Just because you slot on a more powerful GPU doesn't mean the game can utilise it properly hence the lower usage. You can see this kind of behaviour on game with heavy ray tracing being run on AMD GPU. It will show low GPU usage but it is being maxed out, just in a way that isn't utilizing every part of the GPU(RT core obviously arnt being use here on genshin)

**Comment by HadouGun** [comment]
I upgraded to a 7800X3D and most of the places in the game give a solid 200 fps, I have a 4090 and it used to be 120-140 Max and it was heavily dependent on direction.

**Comment by Zeltraa** [comment]
> I upgraded to a 7800X3D and most of the places in the game give a solid 200 fps, I have a 4090 and it used to be 120-140 Max and it was heavily dependent on direction.

What other specs do you have? I have a 3080 with a 5900x and mainly play only genshin right now and I only get 125 in abyss and etc.

**Comment by HadouGun** [comment]
> > I upgraded to a 7800X3D and most of the places in the game give a solid 200 fps, I have a 4090 and it used to be 120-140 Max and it was heavily dependent on direction.
> 
> What other specs do you have? I have a 3080 with a 5900x and mainly play only genshin right now and I only get 125 in abyss and etc.

4k and I use MIGOTO with Nvidia Freestyle Filters, I get 290fps in Abyss. Although I was getting low fps with a bad CPU, it's now good with the CPU upgrade.

**Comment by Zeltraa** [comment]
> > > I upgraded to a 7800X3D and most of the places in the game give a solid 200 fps, I have a 4090 and it used to be 120-140 Max and it was heavily dependent on direction.
> > 
> > 
> > What other specs do you have? I have a 3080 with a 5900x and mainly play only genshin right now and I only get 125 in abyss and etc.
> 
> 4k and I use MIGOTO with Nvidia Freestyle Filters, I get 290fps in Abyss. Although I was getting low fps with a bad CPU, it's now good with the CPU upgrade.

Thanks for the reply! I'll probably upgrade too then, I play in 1440 and I feel like my fps is so low
